test/controllers/api/v2/ostree_branches_controller_test.rb

```diff
@@ -55,5 +55,14 @@ def test_compare
       assert_response :success
       assert_template "katello/api/v2/ostree_branches/compare"
     end
+
+    def test_branch_order
+      @branch2 = @repo.ostree_branches.create!(:name => "def456", :uuid => "456uvw", :version => "1.0")
```

Comment:
- You can make branch2 a local variable since we only use it within this test

- looks like @default_sort is unused so can be removed

- the :version params given to the API are not necessary and I don't think version is a valid parameter - can you tell me what led you that way?

Reply:
Well, my thought was that since we are wanting a list sorted by version, then would we need to include a version for the branch.

Secondly, when I looked at the ostree_branch model, it includes name, uuid, version, version_date and commit. If I'm missing the point, please let me know. Thanks.

Reply:
I see what you mean, but I am referring to current line 63 - I don't think the included version `params` does anything. You're right about wanting to create the branch with a particular version for the purpose of this test, so that should stay